.stylelintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  "extends": [
+    "stylelint-config-standard"
+  ],
+  "customSyntax": "postcss-styled-syntax",
+  "rules": {
+    "no-empty-source": null,
+    "declaration-empty-line-before": null,
+    "alpha-value-notation": "number",
+    "selector-class-pattern": null,
```

Comment:
eventually we'll want to get rid of this but we're still using BEM classes so we need it.